### PR TITLE
Can you add this project help is needed

### PR DESCRIPTION
**Pull request by Dravigon**
Need help to proceed with this project

### CODE REVIEW

File: _data/projects/Qvirinterfave.yml [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+desc: Cross platform Cloud Managemnt tool
+site: https://github.com/Dravigon/Qvirinterfave
+tags:
+- c++
+- qt
+- qml
+- libvirt
+upforgrabs:
+  name: help wanted
+  link: https://github.com/Dravigon/Qvirinterfave
```
Comment (ritwik12) [inline _data/projects/Qvirinterfave.yml]: @Dravigon You need to add issues related to your label "help wanted" and please add the link to your label here.

Comment (Dravigon) [reply _data/projects/Qvirinterfave.yml]: @ritwik12 sure , Well i'm new to this, label do u mean something like a defect label in git

Comment (ritwik12) [reply _data/projects/Qvirinterfave.yml]: Ohhk sure, please go to your repository, then issues and then Labels. Create a label there "help wanted" if it's not already there. Copy that label's url and paste here. Also After creating your label, do mark some issues with that label

Comment (Dravigon) [reply _data/projects/Qvirinterfave.yml]: okay u mean this url https://github.com/Dravigon/Qvirinterfave/labels/help%20wanted
right?
currently i have no tasks in this label but i will do it as soon as possible

Comment (ritwik12) [reply _data/projects/Qvirinterfave.yml]: Yes

Comment (ritwik12) [reply _data/projects/Qvirinterfave.yml]: Also your repo doesn't have any issues. If you want help, you should create some issues for others to contribute and Mark them with this label

Comment (Dravigon) [reply _data/projects/Qvirinterfave.yml]: thanks a lot man :+1:

Comment (Dravigon) [reply _data/projects/Qvirinterfave.yml]: I will create some issues as soon as possible

Comment (ritwik12) [reply _data/projects/Qvirinterfave.yml]: Your welcome

Comment (ritwik12) [reply _data/projects/Qvirinterfave.yml]: Please change the link now

Comment (Dravigon) [reply _data/projects/Qvirinterfave.yml]: is it okay now?

Comment (ritwik12) [reply _data/projects/Qvirinterfave.yml]: Great, now just paste that label link here

Comment (ritwik12) [reply _data/projects/Qvirinterfave.yml]: Anything else you want help with?

Comment (Dravigon) [reply _data/projects/Qvirinterfave.yml]: ok this is the url https://github.com/Dravigon/Qvirinterfave/labels/help wanted

Comment (ritwik12) [reply _data/projects/Qvirinterfave.yml]: Yes, paste this in line 11 of Qvirinterfave.yml

Comment (Dravigon) [reply _data/projects/Qvirinterfave.yml]: okay this url https://github.com/Dravigon/Qvirinterfave/labels/help%20wanted redirects to the label page

Comment (ritwik12) [reply _data/projects/Qvirinterfave.yml]: Yes paste this one in line 11

**Comment by Dravigon** [comment]
@nemequ  done

**Comment by ritwik12** [comment]
@Dravigon Thanks!!

**Pull request merged**